packages/generator-reactserver/.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
Didn't know about this either.  Sweet.

Seems worth putting at the top level rather than down here.